ci/bazelrc_github

```diff
@@ -0,0 +1,34 @@
+# Limit resources since GitHub Actions VMs have 2 cores and 7 GB RAM.
+build --local_cpu_resources=2
+build --local_ram_resources=4000
+build --worker_max_instances=2
+
+# Ensure sandboxing is on to increase hermeticity.
+build --spawn_strategy=sandboxed
+build --worker_sandboxing
+
+# Ensure the PATH env var from our virtualenv propagates into tests, which is
+# no longer on by default in Bazel 0.21.0 and possibly again in the future.
+# We set this flag for "build" since "test" inherits it, but if we don't set
+# it for build too, this causes a rebuild at test time, and if we set it for
+# both we hit https://github.com/bazelbuild/bazel/issues/8237.
+#
+# See also:
+# https://github.com/bazelbuild/bazel/issues/7095 (protobuf PATH sensitivity)
+# https://github.com/bazelbuild/bazel/issues/7026 (future of action_env)
+build --action_env=PATH
+
+# Log more information to help with debugging, and disable curses output which
+# just adds more clutter to the log. (Travis spoofs an interactive terminal.)
```

Comment:
nit: Travis mention here is stale

Reply:
Yeah—I had updated most of the references, but left this one and the
`numpy` business because I didn’t want to bother debugging whether they
were still necessary. But I can try removing them, sure.